modules/reindex/build.gradle

```diff
@@ -37,11 +37,12 @@ run {
 
 dependencies {
   compile "org.elasticsearch.client:rest:${version}"
+  // TODO dm: This is not needed as we pull in dependencies of org.elasticsearch.. projects transitive -> separate PR based on master?
   // dependencies of the rest client
-  compile "org.apache.httpcomponents:httpclient:${versions.httpclient}"
-  compile "org.apache.httpcomponents:httpcore:${versions.httpcore}"
-  compile "commons-codec:commons-codec:${versions.commonscodec}"
-  compile "commons-logging:commons-logging:${versions.commonslogging}"
+  //compile "org.apache.httpcomponents:httpclient:${versions.httpclient}"
+  //compile "org.apache.httpcomponents:httpcore:${versions.httpcore}"
+  //compile "commons-codec:commons-codec:${versions.commonscodec}"
+  //compile "commons-logging:commons-logging:${versions.commonslogging}"
```

Comment:
Note that we need to get that PR in before we can merge this one. Otherwise the build will fail with:

```
Could not resolve all dependencies for configuration ':client:benchmark:compileClasspath'.
> A conflict was found between the following modules:
   - org.apache.httpcomponents:httpcore:4.4.5
   - org.apache.httpcomponents:httpcore:4.4.4
```

The reason is that we use httpcore 4.4.5 wheras httpclient uses 4.4.4. Although transitive dependencies our build plugin disables transitive dependency resolution for all dependencies outside the group "org.elasticsearch".